### PR TITLE
chore(migration): update

### PR DESCRIPTION
**Pull request by xzf0587**

### CODE REVIEW

**Review by jayzhang — DISMISSED** [review]

File: packages/fx-core/src/core/middleware/utils/migrationContext.ts [hunk 1]
```diff
@@ -106,13 +106,21 @@ export class MigrationContext {
...
+  async removeFxV2(): Promise<void> {
+    await this.fsRemove(V2TeamsfxFolder);
   }
 
   async fsPathExists(_path: string): Promise<boolean> {
     return await fs.pathExists(path.join(this.projectPath, _path));
   }
 
+  async fsRemove(_path: string): Promise<void> {
+    return await fs.remove(path.join(this.projectPath, _path));
```
Comment (wenytang-ms) [inline packages/fx-core/src/core/middleware/utils/migrationContext.ts:121]: it's better to check the folder exist, then remove

Comment (xzf0587) [reply packages/fx-core/src/core/middleware/utils/migrationContext.ts:121]: The remove method has existing check inside.

**Review by jayzhang — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

File: packages/fx-core/src/core/middleware/projectMigratorV3.ts [hunk 2]
```diff
@@ -149,8 +158,26 @@ async function rollbackMigration(context: MigrationContext): Promise<void> {
...
+1. Update app id uri in the two manifests
+1. Move .fx/configs/azure.parameter.{env}.json to templates/azure/...
+1. Update placeholders in azure parameter files 
+1. create .env.{env} if not exitsts in teamsfx/ folder (v3) (should throw error if .fx/configs/ not exists?)
+1. migrate .fx/configs/config.{env}.json to .env.{env}
+1. create .env.{env} if not exitsts in teamsfx/ folder (v3)
+1. migrate .fx/states/state.{env}.json to .env.{env}. Skip 4 types of secrets names(should refer to userdata)
+1. create .env.{env} if not exitsts in teamsfx/ folder (v3)
+1. migrate .fx/states/userdata.{env} to .env.{env}
+    `;
```
Comment (wenytang-ms) [inline packages/fx-core/src/core/middleware/projectMigratorV3.ts:177]: why not write this content to a file?

Comment (xzf0587) [reply packages/fx-core/src/core/middleware/projectMigratorV3.ts:177]: Will rewrite this later.

**Pull request merged**